doc/signify.txt

```diff
@@ -653,7 +653,7 @@ The plugin is slow!~
     If you use a decentralized VCS like Git, are you working on a slow remote
     file system?
 
-  * Vim its sign handling code is known to be slow. If the delay is very long,
+  * Vims sign handling code is known to be slow. If the delay is very long,
```

Comment:
Is this even correct? Without an apostrophe it looks like the plural of _Vim_ to me. And I usually try to avoid apostrophes, hence the "long form" using _its_.

Reply:
```suggestion
  * Sign handling in Vim is known to be slow. If the delay is very long,
```